Raven/RavenClient.swift

```diff
@@ -400,7 +383,7 @@ public class RavenClient : NSObject, NSURLConnectionDelegate, NSURLConnectionDat
             }
         }
 
-        #if os(iOS)
+        #if os(iOS) || os(tvOS)
             if (tags["OS version"] == nil) {
```

Comment:
should we add `UIDevice.currentDevice().systemName` to make a difference between tvOS and iOS ? or
`tags["OS version"] = "\(UIDevice.currentDevice().systemName).\(UIDevice.currentDevice().systemVersion)"`